website/src/views/PageStateHandler.ts

```diff
@@ -143,6 +144,8 @@ function toLapisFilterWithoutVariant(
     };
 }
 
+const $ = '$';
```

Comment:
I don't understand this construction. Is this the delimiter?

Reply:
Yes, this is the delimiter. I couldn't find a better name, so I took IntelliJ's suggestion. Apparently `$` is a valid variable name in JS.

Reply:
Can we not name it delimiter?

Reply:
I wanted to be a bit more specific (which delimiter?). If we find a good name, sure. In the meanwhile, I don't find `$` so bad.

Reply:
How about `filterSearchParamDelimiter`?